[improve][doc] SEO for Concepts and Architecture except Overview and Messaging (#674)
apache/pulsar-site: docs/concepts-throttling.md

### Throttling levels

> Configuring throttling levels allows you to throttle message dispatch rate limits at multiple levels. The following table outlines the three throttling levels.

Reviewer: ```suggestion
You can set throttle message dispatch at different levels.
```

### Throttling approaches

> The following table outlines multiple throttling approaches to configure the dispatch rate limits at different levels.

Reviewer: ```suggestion
You can use multiple throttling approaches to configure dispatch rate limits at different levels.
```
